scripts/analysis_1/nbconverted/3_visualize_data.py

```diff
@@ -239,5 +359,56 @@
 combined_data_df = pd.concat([input_data_UMAPencoded_df, simulated_data_UMAPencoded_df])
 
 # Plot
-ggplot(combined_data_df, aes(x='1', y='2')) + geom_point(aes(color='color_by'), alpha=1) + facet_wrap('~dataset') + xlab('UMAP 1') + ylab('UMAP 2') + xlim(7, 11) + ylim(-4, 0) + ggtitle('UMAP of original and simulated data')
+ggplot(combined_data_df, aes(x='1', y='2')) + geom_point(aes(color='color_by'), alpha=1) + facet_wrap('~dataset') + xlab('UMAP 1') + ylab('UMAP 2') + xlim(-2, 3) + ylim(-5,6) + ggtitle('UMAP of original and simulated data (latent space)')
+
+
+# ## Visualize simulated data (latent space) projected into PCA space
+
+# In[35]:
+
+
+# Get and save model
+pca = PCA(n_components=2)
+pca.fit(data_encoded_all_df)
+
+input_data_PCAencoded = pca.transform(data_encoded_all_df)
+input_data_PCAencoded_df = pd.DataFrame(data=input_data_PCAencoded,
+                                         index=data_encoded_all_df.index,
+                                         columns=['1','2'])
+# Add label
+input_data_PCAencoded_df['color_by'] = normalized_data_label['color_by']
+
+ggplot(input_data_PCAencoded_df, aes(x='1',y='2'))     + geom_point(aes(color='color_by'), alpha=1)     + xlab('PCA 1')     + ylab('PCA 2')     + ggtitle('Input data')
+
+
+# In[36]:
+
+
+# Use same UMAP projection to plot simulated data
+simulated_data_PCAencoded = pca.transform(simulated_data_encoded_df)
+simulated_data_PCAencoded_df = pd.DataFrame(data=simulated_data_PCAencoded,
+                                         index=simulated_data.index,
+                                         columns=['1','2'])
+
+# Add back label column
+simulated_data_PCAencoded_df['color_by'] = simulated_data['color_by']
+
+
+ggplot(simulated_data_PCAencoded_df, aes(x='1',y='2'))     + geom_point(aes(color='color_by'), alpha=1)     + xlab('PCA 1')     + ylab('PCA 2')     + ggtitle("Simulated data")
+
+
+# In[38]:
+
+
+# Zoomed in view
+
+# Add label for input or simulated dataset
+input_data_PCAencoded_df['dataset'] = 'original'
+simulated_data_PCAencoded_df['dataset'] = 'simulated'
+
+# Concatenate input and simulated dataframes together
+combined_data_df = pd.concat([input_data_PCAencoded_df, simulated_data_PCAencoded_df])
+
+# Plot
+ggplot(combined_data_df, aes(x='1', y='2')) + geom_point(aes(color='color_by'), alpha=1) + facet_wrap('~dataset') + xlab('PCA 1') + ylab('PCA 2') + ggtitle('PCA of original and simulated data (latent space)')
```

Comment:
hmmm...

Reply:
The plotting was meant to verify that linearly shifting the data in the latent space yields shifting of the data in the original space where the clustering is conserved.  We originally plotted the data using UMAP and we could see most of the clustering is maintained after the shift, but there were some outliers.  So we decided to see what the data looked like using PCA instead since we know that a linear shift should be conserved.  Looking into the UMAP method, there are some assumptions made that would likely cause the outliers.